prisma/schema.prisma

```diff
@@ -13,11 +13,54 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-
 model User {
   id        String   @id @default(uuid()) @db.Uuid
   email     String   @unique
-  name      String?
+  image_url String?
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
-}
\ No newline at end of file
+
+  // Relations
+  Account   Account[]
+}
+
+model Account {
```

Comment:
- Account Should be shared between multiple users
- Use AccountUser as a pivot table to shared multiple users with their respective role